src/Compilers/CSharp/Test/Syntax/Parsing/DeclarationScopeParsingTests.cs

```diff
@@ -139,26 +139,50 @@ public void Method_02(LanguageVersion langVersion)
         public void Method_03(LanguageVersion langVersion)
         {
             string source = "void F(ref scoped int b, in scoped int c, out scoped int d) { }";
-            UsingDeclaration(source, TestOptions.Regular.WithLanguageVersion(langVersion),
-                // (1,19): error CS1001: Identifier expected
-                // void F(ref scoped int b, in scoped int c, out scoped int d) { }
-                Diagnostic(ErrorCode.ERR_IdentifierExpected, "int").WithLocation(1, 19),
-                // (1,19): error CS1003: Syntax error, ',' expected
-                // void F(ref scoped int b, in scoped int c, out scoped int d) { }
-                Diagnostic(ErrorCode.ERR_SyntaxError, "int").WithArguments(",").WithLocation(1, 19),
-                // (1,36): error CS1001: Identifier expected
-                // void F(ref scoped int b, in scoped int c, out scoped int d) { }
-                Diagnostic(ErrorCode.ERR_IdentifierExpected, "int").WithLocation(1, 36),
-                // (1,36): error CS1003: Syntax error, ',' expected
-                // void F(ref scoped int b, in scoped int c, out scoped int d) { }
-                Diagnostic(ErrorCode.ERR_SyntaxError, "int").WithArguments(",").WithLocation(1, 36),
-                // (1,54): error CS1001: Identifier expected
-                // void F(ref scoped int b, in scoped int c, out scoped int d) { }
-                Diagnostic(ErrorCode.ERR_IdentifierExpected, "int").WithLocation(1, 54),
-                // (1,54): error CS1003: Syntax error, ',' expected
-                // void F(ref scoped int b, in scoped int c, out scoped int d) { }
-                Diagnostic(ErrorCode.ERR_SyntaxError, "int").WithArguments(",").WithLocation(1, 54)
-                );
```

Comment:
in some tests *all* syntax diagnostics went away.  as such, i added semantic diagnostics to show we still report problems with the scoped modifier.